src/containers/settings/settings.less

```diff
@@ -0,0 +1,5 @@
+.page-content-container {
```

Comment:
Is there any reason not to use a styled div for this? The same way we use other styled components. I think that'd be better for consistency.

Reply:
No, I was just only using styled components when necessary to override internal styles. It's now updated to use a styled div.